sample/src/test/java/app/cash/paparazzi/sample/MultiplePaparazziTest.kt

```diff
@@ -0,0 +1,49 @@
+package app.cash.paparazzi.sample
+
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import app.cash.paparazzi.Paparazzi
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+class MultiplePaparazziTest {
+
+  @get:Rule
+  val paparazzi = MultiplePaparazziRule()
+
+  @Test(expected = IllegalStateException::class)
```

Comment:
nit: update expected exception with one thrown be `requireNotNull` from suggestion above.